src/services/index.js

```diff
@@ -18,10 +18,10 @@ export const callApi = (endpoint, {
     return json;
   });
 
-export const createDemo = (name, email) =>
+export const createDemo = () =>
   callApi('demos', {
     method: 'POST',
-    body: { name, email },
+    body: { name: 'demo' },
```

Comment:
Correct, the api requires it. You can remove it or make it optional in the api as it shouldn't be needed for our use case.